src/components/events/LiveNowBadge.tsx

```
@@ -0,0 +1,35 @@
+import { cn } from "@/lib/utils";
+
+export interface LiveNowBadgeProps extends React.HTMLAttributes<HTMLDivElement> {
+  showDot?: boolean;
+}
+
+export function LiveNowBadge({
+  className,
+  showDot = true,
+  children,
+  ...props
+}: LiveNowBadgeProps) {
```

ESLint

[error] Replace `⏎··className,⏎··showDot·=·true,⏎··children,⏎··...props⏎` with `·className,·showDot·=·true,·children,·...props·`

(prettier/prettier)